src/Microsoft.AspNetCore.Server.Kestrel.Core/MinDataRate.cs

```diff
@@ -16,9 +16,9 @@ public class MinDataRate
         /// starting at the time data is first read or written.</param>
         public MinDataRate(double bytesPerSecond, TimeSpan gracePeriod)
         {
-            if (bytesPerSecond < 0)
+            if (bytesPerSecond <= 0)
             {
-                throw new ArgumentOutOfRangeException(nameof(bytesPerSecond), CoreStrings.NonNegativeNumberRequired);
+                throw new ArgumentOutOfRangeException(nameof(bytesPerSecond), CoreStrings.PositiveNumberRequired);
```

Comment:
I would like to include something like the following in the exception message. "If you want to disable a minimum rate, use a null MinDataRate instead."